go/vt/mysqlctl/replication.go

```diff
@@ -273,9 +273,12 @@ func (mysqld *Mysqld) ResetReplication(ctx context.Context) error {
 //
 // Array indices for the results of SHOW PROCESSLIST.
 const (
+	//lint:ignore U1000 needed for correct indexing of result columns
```

Comment:
Same comment for all the U1000 suppressions in this PR. Seems like putting those in the codebase doesn't do anything to increase the clarity, maintainability, or correctness of the code.

Reply:
Fixed